pkg/packer/manifests/vsphere/base/packer-base.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Is the plan to also wrap the base image creation in the cli? If so we might want to name these all as templates and then we can embed them and use the existing machinery to write them out with the template input into the work dir at runtime. If not, then we should probably move them out of the go package perhaps to `images/ova/base` so they get captured in the default build / bundle.

Reply:
yes. I am planning to wrap the base image into the cli. The current problem is that it requires the ISO to be uploaded to the vsphere first and then it need to be on same network as vsphere so that the boot process can load kickstart files from packer created HTTP server. There are alternatives to avoid it. I have not got chance to try it yet. I am planning to tackle it in next sprint.